Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -658,19 +658,32 @@ class EditProfileViewController: UIViewController {
     }
 
     @objc private func saveEdit() {
-        let name = nameTextField.getText()?.split(separator: " ")
-        let firstName = String(name?[0] ?? "")
-        let lastName = String(name?[1...].joined(separator: " ") ?? "")
-        let grade = classTextField.getText()
-        let pronouns = pronounsTextField.getText()
-        let phoneNumber = phoneNumTextField.getText()
+        guard let name = nameTextField.getText(), !name.isEmpty,
+              let grade = classTextField.getText(), !grade.isEmpty,
+              let pronouns = pronounsTextField.getText(), !pronouns.isEmpty,
+              let hometown = hometownTextField.getText(), !hometown.isEmpty,
+              let snack = snackTextField.getText(), !snack.isEmpty,
+              let song = songTextField.getText(), !song.isEmpty,
+              let stop = stopTextField.getText(), !stop.isEmpty else {
+                  print("Error: Please complete all fields")
+                  return
+              }
 
-        hometown = hometownTextField.getText()
+        let phoneNumber = phoneNumTextField.getText()
         talkative = talkativeSlider.value
         music = musicSlider.value
-        snack = snackTextField.getText()
-        song = songTextField.getText()
-        stop = stopTextField.getText()
+        self.hometown = hometown
+        self.snack = snack
+        self.song = song
+        self.stop = stop
+
+        guard let nameArray = nameTextField.getText()?.split(separator: " "), nameArray.count == 2 else {
+            print("Error: Name field does not contain first and last name")
```

Comment:
Same as above.

Reply:
I also want to communicate with @rs929 about having the successful print statements for every network request. I was guilty of doing this before, but I now realize that it clogs up the console and makes it difficult to debug, especially since you can have tons of network requests. If you don't see an error message then you can assume that it succeeded.

@tiffany-pann Let me know what you think as well.

Reply:
Missed this notification but I agree, perhaps we can just comment out for now and re-comment if we need to debug networking